### PR TITLE
`General`: include new apollon version in artemis

### PR DESCRIPTION
**Pull request by martindevtum**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Server
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/server/).
- [x] I added multiple integration tests (Spring) related to the features (with a high test coverage).
- [x] I added `@PreAuthorize` and checked the course groups for all new REST Calls (security).
- [x] I implemented the changes with a good performance and prevented too many database calls.
- [x] I documented the Java code using JavaDoc style.
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client-tests/).
- [x] I added `authorities` to all new routes and checked the course groups for displaying navigation elements (links, buttons).
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.
#### Changes affecting Programming Exercises
- [x] I tested **all** changes and their related features with **all** corresponding user types on Test Server 1 (Atlassian Suite).
- [x] I tested **all** changes and their related features with **all** corresponding user types on Test Server 2 (Jenkins and Gitlab).

### Motivation and Context

After updating Apollon to generate both es5 and es6 output, we want to include the new version into Artemis.
Furthermore, we want to test that the new Apollon version is working as expected on an Artemis test server.

### Description

We included the alpha release of the new Apollon version.
We adjusted deep imports of Apollon files in Artemis.

After this PR gets the approvals, we will release an Apollon version and integrate the non-alpha version into Artemis before merging.
### Steps for Testing

Deployed to tests server 5.
1. Go to https://artemistest5.ase.in.tum.de/
2. Navigate into a Modeling exercise
3. Check if Modeling exercises are working correctly

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### CODE REVIEW

**Comment by martindevtum** [comment]
Release and Update Apollon Version after PR has been approved.

- [ ] update version

**Review by bassner — APPROVED** [review]
Can approve as we discussed this fix already! 👍

**Review by willergeorg — APPROVED** [review]
Tested on TS5, worked as expected. Code looks reasonable to me as well. 👍

**Comment by minrows** [comment]
Tested on TS5 and everything looks good to me. 👍

**Review by minrows — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         "@fortawesome/fontawesome-svg-core": "1.3.0",
         "@fortawesome/free-regular-svg-icons": "6.0.0",
         "@fortawesome/free-solid-svg-icons": "6.0.0",
-        "@ls1intum/apollon": "2.9.1",
```
Comment (krusche) [inline package.json:30]: before we merge this, there must be a new version without alpha or beta tag

Comment (krusche) [inline package.json:30]: before we merge this, there must be a new version without alpha or beta tag

**Comment by krusche** [comment]
Not necessary any more, because in another PR the Apollon version was updated to 2.10.0

**Pull request closed without merging**